pkgs/development/libraries/libschrift/default.nix

```
@@ -1,16 +1,22 @@
-{ lib, stdenv, fetchFromGitHub }:
+{ lib, stdenv, fetchFromGitHub, fetchpatch }:
 
 stdenv.mkDerivation rec {
   pname = "libschrift";
-  version = "0.9.1";
+  version = "0.10.0";
 
   src = fetchFromGitHub {
     owner = "tomolt";
     repo = pname;
-    rev = "c6d20460d6e602e8829d3a227fd7be4c4c3cda86";
-    hash = "sha256-BuTmWaWFZ0DXujlbhbmK3Woit8fR9F4DWmKszHX6gOI=";
+    rev = "c207585486b3e78ec5506f55f5d56178f421a53c";
+    sha256 = "13qrplsi2a53s84giwnzqmn0zbslyaagvjn89wsn9fd90m2v2bs1";
   };
 
+  # fix a compilation failure related to darwin integers, remove at next release
+  patches = lib.optional stdenv.isDarwin (fetchpatch {
```

Sorry for another roundtrip but could we apply this patch to all platforms? Otherwise you or someone else might miss removing it.

```suggestion
  patches = (fetchpatch {
```